src/clients/third-party-service-client.ts

```diff
@@ -102,6 +103,13 @@ export const getThirdPartyServiceClient =
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         thirdPartyConfig.prodEnvironment ?? thirdPartyConfig.testEnvironment!;
 
+    eventLog.task.info(`Third party client base url: ${environment.baseUrl}`);
```

Comment:
**Suggestion**: ...also, I would not log the url but just if we are pointing the test environment or not.
Something like:
```suggestion
    eventLog.task.info(`Pointing to test environment? ${thirdPartyConfig.testEnvironment?.testUsers.includes(fiscalCode) ? 'yes' : 'no'}`);
```